code/builders/builder-vite/src/vite-config.ts

```diff
@@ -119,11 +119,5 @@ export async function pluginConfig(options: Options) {
     pluginWebpackStats({ workingDir: process.cwd() }),
   ] as PluginOption[];
 
-  // TODO: framework doesn't exist, should move into framework when/if built
-  if (frameworkName === '@storybook/glimmerx-vite') {
-    const plugin = require('vite-plugin-glimmerx/index.cjs');
-    plugins.push(plugin.default());
-  }
-
```

Comment:
just cleaned up a bit. `@storybook/glimmerx-vite` has never existed, this code path can never been triggered, and it doesn't seem like there's been any movement in glimmerx for 3 years.